src/lwc/gsApplicationStatus/gsApplicationStatus.html

```diff
@@ -0,0 +1,37 @@
+<template >
+    <template if:false={isActiveInstance}>
+        <lightning-card>
+            <div class="spinner" if:true={isLoading}>
+                <lightning-spinner alternative-text="Loading"></lightning-spinner>
+            </div>
+            <div class="slds-grid slds-grid_align-center slds-grid_vertical">
+                <div class="slds-col slds-align_absolute-center">
+                    <img class="image" src={img}/>
+                </div>
+                <div if:false={isApplicationSubmitted} class="slds-col slds-align_absolute-center slds-size_full">
+                    <div class="slds-text-align_center slds-size_full slds-p-around_small">
+                        <div><p class="slds-text-heading_small slds-m-bottom_medium">{labels.gsNoApplicationSubmitted} | <a tabindex="0">{labels.gsLearnMore}</a></p></div>
+                        <div><p class="slds-text-heading_small slds-m-bottom_medium daysLeft">{diffInDays} {labels.gsDaysRemainingInFreeTrial}</p></div>
+                        <div><a tabindex="0" class="slds-button slds-button_brand slds-size_full slds-text-heading_small slds-p-around_medium" href="https://powerofus.force.com/PowerOfUsApply" target="_blank">{labels.gsApplyForFreeLicenses}</a></div>
```

Comment:
You shouldn't need `tabindex` here or up on line 13. What behavior are you seeing that requires special handling?

Reply:
On Chrome without the tabindex I was not reaching the elements using tab key.

Reply:
Is there a spot in the UI where this component is rendering yet, so I can take a look? Thanks!

Reply:
It is a component for a lightning page, there is no page yet though